### PR TITLE
fix: force delete kms key during testing

### PR DESCRIPTION
**Pull request by shemau**
### Description

The complete test should force delete the KMS key.

### Release required?

- [x] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: examples/complete/main.tf [hunk 1]
```diff
@@ -28,7 +28,8 @@ module "key_protect_all_inclusive" {
       key_ring_name = "icd-edb"
```
Comment (ocofaigh) [inline examples/complete/main.tf:28]: can you add `force_delete_key_ring = true` as well?

**Review by ocofaigh — CHANGES_REQUESTED** [review]
@shemau 1 comment. Also pre-commit hook failed

**Comment by shemau** [comment]
```hcl
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185: 
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185: Terraform used the selected providers to generate the following execution
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185: plan. Resource actions are indicated with the following symbols:
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185:   ~ update in-place
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185: 
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185: Terraform will perform the following actions:
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185: 
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185:   # module.key_protect_all_inclusive.module.kms_key_rings["icd-edb"].ibm_kms_key_rings.key_ring will be updated in-place
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185:   ~ resource "ibm_kms_key_rings" "key_ring" {
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185:       ~ force_delete  = false -> true
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185:         id            = "icd-edb:keyRing:crn:v1:bluemix:public:kms:eu-de:a/abac0df06b644a9cabc6e44f55b3880e:be69ec9b-8e84-4459-859a-5276f45fd57f::"
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185:         # (3 unchanged attributes hidden)
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185:     }
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185: 
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185:   # module.key_protect_all_inclusive.module.kms_keys["icd-edb.edb-upg-yxwn2d-edb"].ibm_kms_key.key will be updated in-place
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185:   ~ resource "ibm_kms_key" "key" {
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185:       ~ force_delete            = false -> true
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185:         id                      = "crn:v1:bluemix:public:kms:eu-de:a/abac0df06b644a9cabc6e44f55b3880e:be69ec9b-8e84-4459-859a-5276f45fd57f:key:85ee47dd-0907-4bb4-a223-7828147d383b"
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185:         # (14 unchanged attributes hidden)
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185:     }
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185: 
TestRunUpgradeCompleteExample 2024-06-04T10:50:49Z command.go:185: Plan: 0 to add, 2 to change, 0 to destroy.
```

The change is confined to the complete example used for upgrade testing, related to the KMS module upgrade. Skipping upgrade test.

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**